src/views/UserDetail/UserDetail.js

```diff
@@ -314,6 +316,69 @@ class UserDetail extends React.Component {
     );
   }
 
+  getActionMenu = barcode => ({ onToggle }) => {
+    const showActionMenu = this.props.stripes.hasPerm('ui-users.edit')
+    || this.props.stripes.hasPerm('ui-users.patron_blocks')
+    || this.props.stripes.hasPerm('ui-users.feesfines.actions.all')
+    || this.props.stripes.hasPerm('ui-requests.all');
+
+    const linkToCreateRequest = barcode ?
+      `/requests/?${queryString.stringify({ layer: 'create', userBarcode: barcode })}` :
+      `/requests/?${queryString.stringify({ layer: 'create' })}`;
+
+    if (showActionMenu) {
+      return (
+        <>
+          <IfPermission perm="ui-requests.all">
```

Comment:
This is duplicate to code in _UserForm.js_, as reported by sonarcloud.
Can you please reduce the code duplicity so that your code will pass the quality gate?

Reply:
Well, I de-duplicate the code, do not show the action menu in the create user form anymore and finally I get the necessary code coverage.
Thanks for your feedback @rchr!